src/main/java/taxi/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,254 @@
+package taxi.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import taxi.exception.DataProcessingException;
+import taxi.lib.Dao;
+import taxi.model.Car;
+import taxi.model.Driver;
+import taxi.model.Manufacturer;
+import taxi.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    private static final int ZERO_PLACEHOLDER = 0;
+    private static final int SHIFT = 2;
+
+    @Override
+    public Car create(Car car) {
+        String insertQuery = "INSERT INTO cars (model, manufacturer_id)"
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createCarStatement =
+                        connection.prepareStatement(
+                             insertQuery, Statement.RETURN_GENERATED_KEYS)) {
+            createCarStatement.setString(1, car.getModel());
+            createCarStatement.setLong(2, car.getManufacturer().getId());
+            createCarStatement.executeUpdate();
+            ResultSet resultSet = createCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create car " + car, e);
+        }
+        insertAllDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectQuery = "SELECT c.id as id, "
+                + "model, "
+                + "manufacturer_id, "
+                + "m.name as manufacturer_name, "
+                + "m.country as manufacturer_country "
+                + "FROM cars c"
+                + " JOIN manufacturers m on c.manufacturer_id = m.id"
+                + " where c.id = ? AND c.is_deleted = false";
```

Comment:
if you write queries in upper case then use it everywhere, the same for other queries